ds_judgements_public_ui/sass/includes/_results.scss

```diff
@@ -53,4 +53,10 @@
     padding: $spacer-unit calc($spacer-unit * 2);
     margin: $spacer-unit 0;
   }
+
+  &__list-layout-container {
+    .govuk-warning-text {
+      margin-bottom: 1.5rem;
```

Comment:
Should this be a multiplier of `$spacer-unit`, or is this actually a fixed unit?

Reply:
Updated this :)